docs/setup-moodle.md

```diff
@@ -1,110 +1,100 @@
+1. In the _IMS LTI Names and Role Provisioning_ menu, select **Use this service**.
 
     <div class="d-flex flex-justify-around">
       <img src="/images/help/lms/moodle/step-11.png" class="border" style="width: 30%;">
     </div>
 
-12. Scroll down to the bottom of the page and click “Save”
-
-13. You should now be back at the page which looks like
+1. Scroll down to the bottom of the page and click **Save changes**. You should retun to the page that looks like this:
 
-    <div class="d-flex flex-justify-around">
-      <img src="/images/help/lms/moodle/step-13.png" class="border" style="width: 30%;">
-    </div>
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/moodle/step-13.png" class="border" style="width: 30%;">
+   </div>
 
-14. Ensure that “GitHub Classroom - [Your Classroom Name]” is selected within the “Preconfigured tool” dropdown
+1. In the _Preconfigure tool_ menu, select _GitHub Classroom - [Your Classroom Name]_.
 
-15. Within “Common module settings”, make sure to select Hide from Students for “Availability”
+1. In the _Activity_ menu (under _Common module settings_), select **Hide from students**.
 
-    <div class="d-flex flex-justify-around">
-      <img src="/images/help/lms/moodle/step-15.png" class="border" style="width: 30%;">
-    </div>
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/moodle/step-15.png" class="border" style="width: 30%;">
+   </div>
 
-16. Scroll to the bottom of the page and click “Save and return to course”
+1. Scroll to the bottom of the page and click **Save and return to course**.
 
-17. You should now see a “GitHub Classroom” activity wherever your chose to place it
+1. You should now see a _GitHub Classroom_ activity wherever your chose to place it.
 
     <div class="d-flex flex-justify-around">
       <img src="/images/help/lms/moodle/step-17.png" class="border" style="width: 30%;">
     </div>
 
-18. Click the newly configured GitHub Classroom activity link to launch
+1. Click the GitHub Classroom activity.
 
-19. You should now be directed the following screen. Congratulations! You successfully linked GitHub Classroom to your Moodle.
+You should now be directed the following screen. You have successfully linked GitHub Classroom to your Moodle.
 
-    <div class="d-flex flex-justify-around">
-      <img src="/images/help/lms/moodle/success.png" class="border" style="width: 75%;">
-    </div>
+<div class="d-flex flex-justify-around">
+  <img src="/images/help/lms/moodle/success.png" class="border" style="width: 75%;">
+</div>
 
-### Next Steps
-After successfully linking GitHub Classroom to Moodle, you are now able to use enhanced current and planned capabilities made possible by this rich integration.
+### Next steps
 
-[Importing your course roster from Moodle to GitHub Classroom]()
+[Importing your course roster from your learning management system to GitHub Classroom]().
```

Comment:
This link appears in three places. If there's not already an issue tracking the creation of this content before launching this feature, then we'll need that — this is the only link I found that was broken

Reply:
Could you elaborate a little bit more? I think I'm reading this in the sense that we should use the same text across docs, but I'm a little confused 😅.

Reply:
Sorry, got too aggressive with lining up the text—the change to the text here is a mistake on my part. 😳 I'll push a change to undo that.

The point of my comment was that each of these integration docs ends with this _Next steps_ section, with a link to the importing a course roster docs. Those course roster import docs don't exist yet (and these links don't actually go anywhere). Updating these links should be tracked somewhere, so we don't accidentally ship broken links.

Reply:
Good catch. @thebradbain Do these guides not exist yet or are they just not linked right now? We'll want these guides for launch.

Reply:
Oh yes! Great catch. We should definitely create an issue. I believe our plan before writing those (time permitting, of course) is to see if the insights gained from user testing mandate we make any immediate tweaks to the roster flow, so we don't end up writing docs that are out of date on day one.